pkg/printers/internalversion/printers.go

```diff
@@ -272,6 +272,7 @@ func AddHandlers(h printers.PrintHandler) {
 		{Name: "Name", Type: "string", Format: "name", Description: metav1.ObjectMeta{}.SwaggerDoc()["name"]},
 		{Name: "Capacity", Type: "string", Description: apiv1.PersistentVolumeSpec{}.SwaggerDoc()["capacity"]},
 		{Name: "Access Modes", Type: "string", Description: apiv1.PersistentVolumeSpec{}.SwaggerDoc()["accessModes"]},
+		{Name: "Volume Mode", Type: "string", Description: apiv1.PersistentVolumeSpec{}.SwaggerDoc()["volumeMode"]},
```

Comment:
I guess the column will be blank if the feature is not enabled.  Not sure if there's an easy way to only include the column with a feature gate.

Reply:
@msau42 
I tried `utilfeature.DefaultFeatureGate.Enabled(features.BlockVolume)` feature gate check, but it didn't work. Also I couldn't find example that uses feature gate at existing code under cmdline dir. At least, I believe we need a way/API to obtain feature gate enablement/disablement status via API.